code/__DEFINES/atom_hud.dm

```diff
@@ -8,7 +8,8 @@
 #define WANTED_HUD					"wanted_hud" // wanted, released, parroled, security status
 #define XENO_EMBRYO_HUD				"xeno_embryo_hud" // xeno larval stage.
 #define HEALTH_HUD_XENO				"xeno_health_hud" //health HUD for xenos
-#define SQUAD_HUD					"squad_hud" //squad hud showing who's leader, corpsman, etc for each squad.
+#define SQUAD_HUD_LOYALIST			"squad_hud_loyalist" //squad hud showing who's leader, corpsman, etc for each squad.
```

Comment:
loyalist what ? i think tgmc is better
```suggestion
#define SQUAD_HUD_TGMC		"squad_hud_tgmc" //squad hud showing who's leader, corpsman, etc for each squad.
```

Reply:
Not tgmc just use terragov like all the other defines